plugins/out_s3/s3.c

```diff
@@ -1350,16 +1553,14 @@ static void cb_s3_flush(const void *data, size_t bytes,
                             struct flb_config *config)
```

Comment:
As you rename some variables in this function, can you please also update the comments and add more comments for update logic in it?